utils/hare-bootstrap

```diff
@@ -53,6 +50,29 @@ say() {
     echo -n "$(date '+%F %T'): $*"
 }
 
+ssh_safe() {
```

Comment:
suggestion: s/ssh_safe/ssh_error_info ( we are not doing anything from security perspective as such :))

Reply:
yeah. This seems much better. I'll change it to ssh_error_info

Reply:
Done.